.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -2,25 +2,6 @@ name: Bug report
 description: Use this template for reporting bugs. Please search existing issues first.
 labels: bug, needs-triage
 body:
-  - type: textarea
-    attributes:
-      label: Brief summary
-    validations:
-      required: true
```

Comment:
I removed brief summary as it's often already included in the title, but I'm open to making it an optional field